.github/workflows/ci-cd-staging.yaml

```
@@ -43,6 +43,7 @@ jobs:
       VITE_SENTRY_DSN: ${{ secrets.VITE_SENTRY_DSN }}
     uses: ./.github/workflows/run-ci-cd.yaml
     with:
+      aws-region: us-east-1
```

actionlint (1.7.12)

[error] input "aws-region" is not defined in "./.github/workflows/run-ci-cd.yaml" reusable workflow. defined inputs are "backend_gid", "backend_uid", "backend_use_fargate_spot", "base_url", "django_configuration", "django_settings_module", "enable_additional_parameters", "enable_cron_tasks", "enable_nat_gateway", "enable_rds_proxy", "environment", "frontend_use_fargate_spot", "release_tag", "tasks_use_fargate_spot"

(workflow-call)